tests/app/views/contexts/test_section_summary_context.py

```diff
@@ -368,6 +368,7 @@ def test_context_for_section_summary_with_list_summary_and_first_variant(
                                                             "unit": None,
                                                             "unit_length": None,
                                                             "value": 123,
+                                                            "decimal_places": None,
```

Comment:
I find it odd that `"decimal_places": None` has been added everywhere, can it be taken out now that things have been fixed? or was this needed?

Reply:
They seem to be still needed as they are part of the generated context